public/js/monitor.js

```diff
@@ -114,25 +114,9 @@ function enableBtnIfEmailValid(e) {
   }
 }
 
-function enableBtnIfInputEmpty(e) {
-  const thisForm = e.target.form;
-  if(thisForm.querySelector(".submit-email")) {
-    if (!e.target.value) {
-      thisForm.classList.remove("invalid");
-
-    } else {
-      enableBtnIfEmailValid(e);
-    }
-  }
-}
-
-function showMessageIfDisabled() {
-  const emailBtns = document.querySelectorAll(".submit-email");
-  for (const emailBtn of emailBtns) {
-    if (emailBtn.disabled) {
-      addClass(".email-form", "invalid");
-    }
-  }
+function showMessageIfDisabled(e) {
+  const thisForm = e.target.parentElement.parentElement;
```

Comment:
This feels risky. Like, it only works because the current target element is wrapped in a `<div>` or something, and if we modify nesting slightly, this will subtly break.
Not sure if there is an easier way to get a parent form wrapper or not.